patterns/vpc-lattice/cross-cluster-pod-communication/cluster/charts/platform/templates/kyverno-cluster-policy.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: inject-sidecar
+  annotations:
+    policies.kyverno.io/title: Inject Sidecar Container
+spec:
+  rules:
+    - name: inject-sidecar
+      match:
+        any:
+          - resources:
+              kinds:
+                - Deployment
+      mutate:
+        patchStrategicMerge:
+          spec:
+            template:
+              metadata:
+                annotations:
+                  (vpc-lattices-svcs.amazonaws.com/agent-inject): "true"
+              spec:
+                initContainers: # IPTables rules are updated in init container
+                  - image: public.ecr.aws/d2c6w7a3/iptables
```

Comment:
Can we use a public image? Such as https://hub.docker.com/r/istio/iptables?

Reply:
I missed this comment, I will try with public image